flow/queue/server_websocket.py

```diff
@@ -0,0 +1,216 @@
+import asyncio
+import websockets
+import util
+import db_functions
+import exceptions
+import datetime
+from typing import Dict, List, Set
+
+
+class Group:
+	def __init__(self, group_id: str, server: 'QueueWebsocketServer'):
+		self.group_id = group_id
+		self.server = server
+		self.listeners = set()  # type: Set[ConnectedClient]
+		self.queue = []  # type: List[QueueTicket]
+
+	async def add_listener(self, listener):
```

Comment:
@imasquid123 In a really simple way, every function that has the async keyword must be called with `await` (eg: `await foobar()`)

These are asynchronous functions, read about them here http://stackabuse.com/python-async-await-tutorial/

Another recommended read is http://stackoverflow.com/questions/231767/what-does-the-yield-keyword-do-in-python/231855#231855